src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -537,12 +536,6 @@ async function ProcessHTTPS(frb: FunctionsRuntimeBundle, trigger: EmulatedTrigge
         res.on("finish", () => {
           instance.close();
           resolveEphemeralServer();
-
-          // If we're on a Unix platform, then the pipe is not cleaned up automatically so...
-          if (process.platform !== "win32") {
```

Comment:
Did you mean to remove this? Doesn't seem like it's relevant to the PR...

Reply:
I did, I just tossed it in there because it was turns out to not actually be needed (it was merged in with another fix this week)